front_end/src/components/history/HistoryOverview.js

```diff
@@ -269,6 +307,37 @@ class HistoryOverview extends Component {
                 <Grid item className="chipsAndTotal">
                   <br />
                   {displayRegularChip}
+                  <Dialog
+                    fullWidth={
+                      width === "md" || width === "lg" || width === "sm"
+                        ? true
+                        : false
+                    }
+                    fullScreen={width === "xs" ? true : false}
+                    open={this.state.open}
+                    onClose={this.handleClose}
+                  >
+                    <DialogTitle>Are you sure you want to cancel?</DialogTitle>
+                    <p className="supportTextDialog">
+                      *Full refund only when cancellation is greater than 48
+                      hours before check in time.
+                    </p>
```

Comment:
It'd be great to have feedback to the user whether this policy will affect him/her.

If we could tell them right away whether they are within the 48 hours period or not.

Reply:
Yep, thanks for the suggestion. Ideally it will be great but I've been facing problems fetching the start time for each unique booking, to do the calculations. Somehow, it is only able to grab the latest start time from the array of bookings and the policy prompt will be based on that for all.

Reply:
Implemented :)